/**
 * Should-fail test for VSGC-2949 : Range step goes to zero - runtime divide by zero
 *
 * Eager Integer case
 *
 * @test
 * @run
 */

import java.lang.Exception;

var c = 0;
try {
  var rs = bind [1..4 step c];
  rs;
  println("BAD");
} catch (exc : Exception) {
  println(exc);
}
